web/src/buddy_list.js

```diff
@@ -55,16 +86,208 @@ class BuddyListConf {
 
 export class BuddyList extends BuddyListConf {
     all_user_ids = [];
+    users_matching_view_ids = [];
+    other_user_ids = [];
+    tooltips_initialized = false;
+
+    initialize_tooltips() {
+        $("#right-sidebar").on("mouseenter", ".buddy-list-heading", (e) => {
+            e.stopPropagation();
+            const $elem = $(e.currentTarget);
+            let placement = "left";
+            if (window.innerWidth < media_breakpoints_num.md) {
+                // On small devices display tooltips based on available space.
+                // This will default to "bottom" placement for this tooltip.
+                placement = "auto";
+            }
+            const subscriber_count_function = this.total_subscriber_count;
+            tippy($elem[0], {
+                // Because the buddy list subheadings are potential click targets for purposes
+                // having nothing to do with the subscriber count (collapsing/expanding), we
+                // delay showing the tooltip until the user lingers a bit longer.
+                delay: [300, 0],
```

Comment:
Rather than hardcode this value, it would be better to use something like the `INTERACTIVE_HOVER_DELAY` constant from `tippyjs.ts`. I think there's a goal to have a relatively small set of delays that are used uniformly throughout the UI.

Reply:
Sure, I'll try that! For the record @alya this means that the tooltip would show up after 425ms of hover and go away 20ms after hover ends.